src/common/config.cc

```diff
@@ -786,45 +811,30 @@ int md_config_t::_get_val(const char *key, char **buf, int len) const
   // In key names, leading and trailing whitespace are not significant.
```

Comment:
The above -EINVAL is wrong since it looks like a valid integer option from the caller point of the view